protobuf_util.go

```diff
@@ -22,8 +22,8 @@ import (
 // corresponding to the Protobuf package, service, and method. It always starts
 // with a slash. Within connect, we use this as (1) Spec.Procedure and (2) the
 // path when mounting handlers on muxes.
-func extractProtoPath(url string) string {
-	segments := strings.Split(url, "/")
+func extractProtoPath(path string) string {
```

Comment:
This was mildly interesting in that now passing explicitly the url.Path here resolves a minor bug in this logic.

Before, a full url was passed into here such as `http://localhost/foo`, which ends up being parsed down to `/localhost/foo` rather than just simply `/foo`.

I doubt in practice this was an issue, but this felt more correct.